From: =?utf-8?b?T25kxZllaiBOb3bDvQ==?= <onovy@debian.org>
Date: Mon, 1 Oct 2018 11:43:03 +0200
Subject: Make the references reproducible

Forwarded: https://bitbucket.org/zzzeek/changelog/pull-requests/1/please-make-the-references-reproducible/
---
 changelog/render.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/changelog/render.py b/changelog/render.py
index 3ec87e2..2e3e7e4 100644
--- a/changelog/render.py
+++ b/changelog/render.py
@@ -171,7 +171,7 @@ def _render_rec(changelog_directive, rec, section, cat, append_sec):
             (rec['changeset'],
              changelog_directive.env.config.changelog_render_changeset, "r%s"),
     ):
-        for refname in collection:
+        for refname in sorted(collection):
             if i > 0:
                 insert_ticket.append(nodes.Text(", ", ", "))
             else:
